.rubocop.yml

```diff
@@ -31,3 +31,9 @@ Lint/BlockAlignment:
 
 Lint/InheritException:
   Enabled: false
+
+Style/GuardClause:
+  Enabled: false
```

Comment:
you don't like it telling you to use guard clauses? I like guard clauses in ruby